src/components/SearchPage.js

```diff
@@ -77,8 +87,9 @@ class SearchPage extends React.Component {
     retrieveTableData() {
 
         let sequenceTime = new Date();
-        let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
-                  '&word=' + this.state.word;
+
+		let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
+				  '&word=' + this.state.word + "&pageRow=" + this.state.rowsPerPage;
```

Comment:
word can sometimes be empty, wat happens then? default for pagerow is 10 anyways?